### PR TITLE
[amqpcpp/leptonica/glibmm] Update to latest version

### LINKED ISSUE #19969: [glibmm] update to 2.68.1
Library name: glibmm

New version number: 2.68.1

Other information that may be useful (release notes, etc...)
https://download.gnome.org/sources/glibmm/2.68/

### LINKED ISSUE #19970: [leptonica] update to 1.81.1
Library name: leptonica

New version number: 1.81.1

Other information that may be useful (release notes, etc...)
https://github.com/DanBloomberg/leptonica/releases

### LINKED ISSUE #19971: [AMQP-CPP] update to 4.3.14
Library name: AMQP-CPP

New version number: 4.3.14

Other information that may be useful (release notes, etc...)
https://github.com/CopernicaMarketingSoftware/AMQP-CPP

### PR DESCRIPTION
**Pull request by Cheney-W**
**Describe the pull request**

- #### What does your PR fix?  
  Fixes https://github.com/microsoft/vcpkg/issues/19971
  Fixes https://github.com/microsoft/vcpkg/issues/19970
  Fixes https://github.com/microsoft/vcpkg/issues/19969

No feature need to be tested.

### CODE REVIEW

**Review by NancyLi1013 — APPROVED** [review]

File: ports/amqpcpp/vcpkg.json [hunk 1]
```diff
@@ -1,10 +1,18 @@
 {
   "name": "amqpcpp",
-  "version-string": "4.3.11",
+  "version-semver": "4.3.14",
```
Comment (BillyONeal) [inline ports/amqpcpp/vcpkg.json:3]: I'm pretty sure "version" was correct. Did upstream indicate they follow semver?

File: ports/glibmm/vcpkg.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "glibmm",
-  "version": "2.68.0",
+  "version-semver": "2.68.1",
```
Comment (BillyONeal) [inline ports/glibmm/vcpkg.json:3]: Ditto is changing to semver correct?

**Review by BillyONeal — CHANGES_REQUESTED** [review]

File: ports/amqpcpp/vcpkg.json (hunk 1 above)
Comment (Cheney-W) [reply ports/amqpcpp/vcpkg.json]: Thanks for pointing it out. I have modified the version schemes.

File: ports/glibmm/vcpkg.json (hunk 2 above)
Comment (Cheney-W) [reply ports/glibmm/vcpkg.json]: Modified the version schemes, thank you!

**Review by NancyLi1013 — APPROVED** [review]

**Comment by BillyONeal** [comment]
Thanks for the updates!

**Pull request merged**